src/manager/PseudoStateTool.tsx

```diff
@@ -1,5 +1,6 @@
 import React, { useCallback } from "react"
-import { Icons, IconButton, WithTooltip, TooltipLinkList } from "@storybook/components"
```

Comment:
Is this the only declaration of `Icons`? Does [this line](https://github.com/chromaui/storybook-addon-pseudo-states/pull/100/files#diff-a0545e75046f4ff07d0014505213b181ffe6cc6b01e916c0545e5f54d7b60581R65) need updated as well?

Reply:
Yeah 👍